src/components/Careers/WhyWereHere/index.tsx

```diff
@@ -12,16 +12,16 @@ export const WhyWereHere = () => {
                             <div>
                                 <p>
                                     Until now, tools for building products have been fragmented. Product analytics,
-                                    heatmaps, session recording and A/B testing all provide immense value, but existing
-                                    tools weren’t natively built to work together.
+                                    heatmaps, session recording, feature flagging and A/B testing are all helpful, but
```

Comment:
nit but... feature flagging -> feature flags, just based on the fact we use that phrase everywhere else

I get the "-ing" thing, but we don't say "heatmapping"